demos/python_demos/asl_recognition_demo/README.md

```diff
@@ -73,9 +73,11 @@ python asl_recognition_demo.py \
 -c ./classes.json
 ```
 
+The demo starts in person tracking mode and to switch it in the action recognition mode you should press `0-9` button with appropriate detection ID (the number in top-left of each bounding box). After that you can switch back to tracking mode by pressing space button.
```

Comment:
Space usually says for pause. I suggest Tab.

Reply:
IMHO, the `space` button is big and attractive for pressing and also it doesn't require to aim on it :-).

Reply:
OK